libs/entry-components/styles/partials/core/components/tables/_layout.scss

```diff
@@ -1,22 +1,8 @@
 @use '@enigmatry/scss-foundation/src/modules/display/items';
 
-.entry-table {
-	display: flex;
-	position: relative;
-	flex-direction: column;
-	width: 100%;
-
-	.mat-mdc-table-container {
-		overflow: auto;
-
-		&.mat-mdc-table-with-data {
-			flex: 1;
-		}
-	}
-
-	.mat-mdc-table {
-		min-width: 100%;
-		border-collapse: separate;
+.mat-table-container {
```

Comment:
Please leave table wrapper (or in case of other components, other entry- prefix on the first class) since this might have more effect than we'd like.